File: Interfaz/ProyectoFinal/src/app/Usuario/class/usuario.ts
export class Usuario {
    private Nombre: string;

    private Apellido: string;


    constructor()
    {
        this.Nombre = '';
        this.Apellido = '';
    }

    public get nombre(): string {
        return this.Nombre;
    }

    public set nombre(value: string) {
        this.Nombre = value;
    }

    public get apellido(): string {
        return this.Apellido;
    }
    public set apellido(value: string) {
        this.Apellido = value;
    }
}

File: ProyectoFinal/ProyectoFinal/2_Dominio/Entidades/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal._2_Dominio
{
    public class Usuario
    {
        public int IdUsuario { get; set; }

        public string Nombre { get; set; }

        public string Apellido { get; set; }

        public string Contrasena { get; set; }

        public string Mail { get; set; }

        public string Cedula { get; set; }

        public DateTime FechaDeIngreso { get; set; }

        public DateTime FechaDeModificacion { get; set; }

        public bool Activo { get; set; }
    }
}